timemachine/cpp/src/kernels/k_flat_bottom_bond.cuh

```diff
@@ -38,9 +38,9 @@ void __global__ k_log_probability_selection(
     RealType atom_atom_dy = coords[idx * 3 + 1] - coords[reference_idx * 3 + 1];
     RealType atom_atom_dz = coords[idx * 3 + 2] - coords[reference_idx * 3 + 2];
 
-    atom_atom_dx -= bx * nearbyint(atom_atom_dx * inv_bx);
-    atom_atom_dy -= by * nearbyint(atom_atom_dy * inv_by);
-    atom_atom_dz -= bz * nearbyint(atom_atom_dz * inv_bz);
+    atom_atom_dx -= bx * nearbyintf(atom_atom_dx * inv_bx);
+    atom_atom_dy -= by * nearbyintf(atom_atom_dy * inv_by);
+    atom_atom_dz -= bz * nearbyintf(atom_atom_dz * inv_bz);
```

Comment:
nit: we may not really care, but would this induce a narrowing conversion when `RealType` is `double`? Would it be worth adding a static conditional to use `nearbyint` in the latter case?

Reply:
Since it didn't impact any of our float64 tests it seemed unnecessary, though might be too trusting of the tests. But there is the question of whether we want to avoid any narrowing conversions, which would be a reasonable convention.